fixtures/CAPI.ts

```diff
@@ -8,6 +8,15 @@ export const CAPI: CAPIType = {
     designType,
     webPublicationDate: '2018-11-02T09:45:30.000Z',
     webPublicationDateDisplay: 'Fri 02 Mar 2018 09.45 GMT',
+    pageType: {
+        hasShowcaseMainElement: false,
+        isFront: false,
+        isLiveblog: false,
+        isMinuteArticle: false,
+        isPaidContent: false,
+        isPreview: false,
+        isSensitive: false,
+    },
```

Comment:
I know that CAPI has this as a separate object but I can't see any good reason to do this so maybe we can flatten these properties when we bring them into DCR?

Reply:
As discussed irl, even though your suggestion would make sense for DCR, it's slightly outside of the scope of what this PR is supposed to do, so my preference is to use the `pageType` as is and consider a refactoring task in the DCR backlog.